app/views/feedback/create.html.haml

```diff
@@ -0,0 +1,3 @@
+Thanks for your feedback
+
+= link_to "Return to gov.au", @feedback.url
```

Comment:
Now that I think about it some more, isn't it normal to redirect after a post? So the user doesn't resubmit the form if they refresh? So maybe this should be the show view rather than the create view.